### PR TITLE
Mazda updates

### PR DESCRIPTION
**Pull request by Jafaral**

### CODE REVIEW

File: mazda_2017.dbc [hunk 1]
```diff
@@ -701,6 +707,12 @@ BO_ 1143 BSM: 8 XXX
  SG_ NEW_SIGNAL_1 : 32|1@0+ (1,0) [0|1] "" XXX
  SG_ REAR_CT_ALERT : 23|5@0+ (1,0) [0|63] "" XXX
 
+BO_ 480 ACCEL_TEST: 8 XXX
+ SG_ ACCEL_COMMAND : 7|32@0- (1,0) [-2147483647|2147483647] "" XXX
+ SG_ ENABLED : 32|1@0+ (1,0) [0|1] "" XXX
+ SG_ STARTING : 40|1@0+ (1,0) [0|1] "" XXX
+ SG_ STOPPING : 48|1@0+ (1,0) [0|1] "" XXX
```
Comment (sshane) [inline mazda_2017.dbc:714]: What is this?

File: mazda_2017.dbc [hunk 2]
```diff
@@ -554,14 +556,18 @@ BO_ 535 CURVE_CTRS: 8 XXX
 
 BO_ 540 CRZ_CTRL: 8 XXX
  SG_ NEW_SIGNAL_6 : 10|1@0+ (1,0) [0|1] "" XXX
+ SG_ NEW_SIGNAL_9 : 31|1@0+ (1,0) [0|255] "" XXX
```
Comment (sshane) [inline mazda_2017.dbc:559]: What are all the NEW_SIGNALs?

**Comment by adeebshihadeh** [comment]
@Jafaral any updates on this?

**Comment by Jafaral** [comment]
> @Jafaral any updates on this?

Will have to look at drives to remember, but one part is about the TJA feature in some newer models, and I was also probably digging into the radar signals at the time. I have been busy in recent  months so I didn't get a chance to get back to this.

**Comment by adeebshihadeh** [comment]
@sshane as long as it doesn't affect any current openpilot behavior, seems fine to merge?

**Pull request merged**